samples/sensor/bmi270/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020 Bosch Sensortec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0 {
```

Comment:
Please do not limit it to nrf52840dk, use &arduino_i2c and rename file to samples/sensor/bmi270/boards/app.overlay

Reply:
Hi @jfischer-no, I tried this, but it broke the device tree enumeration. Are there other files that need to be modified too? I suppose the sample.yaml?

Reply:
samples/sensor/bmi270/boards/app.overlay:
```c
&arduino_i2c {
	status = "okay";

	bmi270@68 {
		compatible = "bosch,bmi270";
		reg = <0x68>;
		label = "BMI270";
	};
};
```
and

```diff
diff --git a/samples/sensor/bmi270/sample.yaml b/samples/sensor/bmi270/sample.yaml
index bcabd60b6b..884fe164d9 100644
--- a/samples/sensor/bmi270/sample.yaml
+++ b/samples/sensor/bmi270/sample.yaml
@@ -4,4 +4,4 @@ tests:
   sample.sensor.bmi270:
     harness: sensor
     tags: samples sensor
-    platform_allow: nrf52840dk_nrf52840
+    depends_on: arduino_i2c
```

should work, but if you have trouble let it how it is for now.

Reply:
Hi @jfischer-no, 
This change is breaking the buildkite build. It works on my system provided the name of overlay file is not changed and left at nrf52840dk_nrf52840. This is part of the current pull-request. I would need to revert the dependency in the sample.yaml file to have a successful build. Please advise.

Reply:
@BoschSensortec You need to move/rename `samples/sensor/bmi270/boards/nrf52840dk_nrf52840.overlay -> samples/sensor/bmi270/app.overlay`. You can fix it by doing rebase on parent commit 70dc14cfc7962eabf0e9387db4cadf5ddbee559d, for example:
```bash
git rebase -i 70dc14cfc7962eabf0e9387db4cadf5ddbee559d
git mv samples/sensor/bmi270/boards/nrf52840dk_nrf52840.overlay samples/sensor/bmi270/app.overlay
git commit --amend --no-edit
git rebase --continue
```

Reply:
Hi @jfischer-no , Thanks for the clarification. Seems that I missed the detail about the file not being the in the boards directory.